mp/src/game/shared/momentum/mom_gamerules.cpp

```diff
@@ -439,6 +439,16 @@ void CMomentumGameRules::ApplyRadiusDamage(CBaseEntity *pEntity, const CTakeDama
     else
     {
         pEntity->TakeDamage(adjustedInfo);
+        if (pEntity->IsPlayer())
+        {
+            CMomentumPlayer *pPlayer = CMomentumPlayer::GetLocalPlayer();
```

Comment:
Use `static_cast<CMomentumPlayer*>(pEntity)` instead of getting the local player here.